README.md

```diff
@@ -17,16 +19,32 @@ Mirecka J, Famili M, Kotanska A, Jurashcko N, Costa-Gomes B, Palmer CM, Thiyagal
 
 > Note: This has been tested in the `refactor` branch.
 
-You can install the libraries needed for this package on a [fresh virtual environment](https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/) with the following:
+You can install the libraries needed for this package on a
+[fresh virtual environment](https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/)
+with the following:
 
 ```
 python3 -m venv env
 source env/bin/activate
-pip install -e .
+python -m pip install -e .
```

Comment:
Since the `venv` is running from `python3` endpoint maybe update this to `pyton3` too? or set both to `python` for consistency.